voxblox/include/voxblox/integrator/esdf_occ_integrator.h

```diff
@@ -38,8 +38,7 @@ class EsdfOccIntegrator {
   // Fixed is overloaded as occupied in this case.
   // Only batch operations are currently supported for the occupancy map.
   void updateFromOccLayerBatch();
-  void updateFromOccBlocks(const BlockIndexList& occ_blocks,
-                           bool push_neighbors);
```

Comment:
@helenol this push_neighbor was never used so I removed it, let me know if this is ok or if it should have been used and this was a bug.

Reply:
I think no problem to remove, probably meant to implement it at some point but clearly didn't.